### PR TITLE
add folders + update README.md

### PR DESCRIPTION
**Pull request by hritikaggarwal2**
Added new folders and README.md files to separate out code based on their functionality and use.

### CODE REVIEW

File: web/src/screens/_README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Screens
+
+## Why this folder?
+
+- Add complete screen components in this folder. Things like Dashboard, Profile
```
Comment (thornegarvin) [inline web/src/screens/_README.md:5]: Might go through and add "do not put screen components here, strictly widgets, etc." elsewhere

File: web/src/components/_README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Components
+
+## Why this folder?
+
+- Add individual components in this folder. Things like FancyButton, SearchBar,
```
Comment (thornegarvin) [inline web/src/components/_README.md:5]: Might be too vague - would specify "this is for widgets that do not take up the full page", etc. Whatever you want

**Review by thornegarvin — COMMENTED** [review]
Might be more specific in a few places, and drop some notes as to what NOT to include

**Review by chrisyuen17 — COMMENTED** [review]
Is there a reason why that commit was reverted?

**Comment by hritikaggarwal2** [comment]
> Is there a reason why that commit was reverted?
Those changes do not fall under this comment so I thought we can separate them. Will do another push with those changes once this branch has been merged

**Review by chrisyuen17 — APPROVED** [review]

**Pull request merged**